dns/zone-configs/k8s.io.yaml

```diff
@@ -248,3 +248,7 @@ kind.sigs:
 cluster-api.sigs:
   type: CNAME
   value: kubernetes-sigs-cluster-api.netlify.com.
+# https://github.com/kubernetes/cloud-provider-vsphere docs (@andrewsykim, @frapposelli)
+vsphere:
+  type: CNAME
+  value: kubernetes-on-vsphere.netlify.com
```

Comment:
This needs to follow the process laid out in https://github.com/kubernetes/community/blob/master/github-management/subproject-site-requests.md

Reply:
We've also been suggesting projects to use sigs.k8s.io rather than k8s.io, even if they are in the main kubernetes org

Reply:
ah, this is good to know! thanks @nikhita!

Reply:
This needs to be hosted on the upstream netlify instance/team. If you can create an issue on k/org about this and assign it to me, I can set that up :)

Reply:
The netlify URL would be `kubernetes-sigs-cloud-provider-vsphere.netlify.com`